packages/server/graphql/getValidTeamIds.ts

```diff
@@ -0,0 +1,8 @@
+export const getValidTeamIds = (teamIds: null | string[], tms: string[]) => {
+  // the following comments can be removed pending #4070
+  // const viewerTeamMembers = await dataLoader.get('teamMembersByUserId').load(viewerId)
+  // const viewerTeamIds = viewerTeamMembers.map(({teamId}) => teamId)
+  if (teamIds?.length) return teamIds!.filter((teamId) => tms.includes(teamId))
```

Comment:
+1 this isn't your code, but we can remove the `!` from `teamIds!` as we're checking if `teamIds` exists